django/chat/templates/chat/components/stop_button.html

```diff
@@ -7,6 +7,6 @@
           hx-get="{% url 'chat:stop_response' message_id=message.id %}"
           hx-swap="none">
     <i class="bi bi-stop-circle"></i>
-    <span class="d-none d-md-inline">{% trans "Stop" %}</span>
+    <span class="d-none d-md-inline">{% trans "Stop Output <br> WARNING: stopping may not reduce costs" %}</span>
```

Comment:
Previously this said "_will_ not reduce costs", but that's not technically true as there are some circumstances where you can actually reduce costs if you're fast enough (see above)